module/FileNVRAM.c

```diff
@@ -359,7 +388,18 @@ static void readplist()
                 
                 if (plistSize && read(fh, plistBase, plistSize) == plistSize)
                 {
-                    XMLParseFile( plistBase, &gPListData );
+                    // cleaning nvram.plist from \n and \t.
+                    // data tag have those bytes after the serialization:
+                    //
+                    //  <key>SystemAudioVolume</key>
+                    //  <data>
+                    //      fw==
+                    //  </data>
+                    //
+                    // but I want it to be: <data>fw==</data>
+                    //
+                    // so some garbage ends up in /chosen/nvram. just remove them!
+                    XMLParseFile(strdelchar(strdelchar(plistBase, '\n'), '\t'), &gPListData);
```

Comment:
I think is there for years. XMLCastData is only used by this module. Maybe I can do that here, but a (normal) plist file really contains end of line and tabs under the data tag.